src/rs_driver/driver/decoder/member_checker.hpp

```diff
@@ -137,3 +137,67 @@ inline typename std::enable_if<RS_HAS_MEMBER(T_Point, feature)>::type setFeature
   point.feature = value;
 }
 
+template <typename T_Point>
+inline typename std::enable_if<!RS_HAS_MEMBER(T_Point, feature)>::type setYaw(T_Point& point,
```

Comment:
```suggestion
inline typename std::enable_if<!RS_HAS_MEMBER(T_Point, yaw)>::type setYaw(T_Point& point,
```
similarly to all the other new functions

Reply:
Done and fixed all the other members as well.